pytensor/tensor/slinalg.py

```diff
@@ -639,7 +639,7 @@ def make_node(self, A):
             )
 
         LU = matrix(shape=A.type.shape, dtype=A.type.dtype)
-        pivots = vector(shape=(A.type.shape[0],), dtype="int64")
+        pivots = vector(shape=(A.type.shape[0],), dtype="int32")
```

Comment:
Like why 64 above but 32 here?

Reply:
This is what the scipy function returns. The reason I went with i64 above is that `np.argsort` returns that regardless of the input, so I kept the outputs the same type regardless of `return_inverse` or whatever the property on that other Op is